File: app/src/main/java/com/botijasoftware/utils/interpolator/AccelerationInterpolator.java
package com.botijasoftware.utils.interpolator;

public class AccelerationInterpolator implements Interpolator {
	//public abstract Interpolator() {}
	public float interpolate(float v0, float v1, float t) {
		float a = t*t;
		return v0 + a*(v1 - v0);
	}
}

File: app/src/main/java/com/botijasoftware/utils/interpolator/AccelerationInterpolator.kt
package com.botijasoftware.utils.interpolator

class AccelerationInterpolator : Interpolator {
    //public abstract Interpolator() {}
    override fun interpolate(v0: Float, v1: Float, t: Float): Float {
        val a = t * t
        return v0 + a * (v1 - v0)
    }
}
